screens/ConditionsResults.js

```diff
@@ -27,7 +27,7 @@ const ConditionsResults = ({ route }) => {
 
   const fetchConditionsForLure = async () => {
     const response = await fetch(
-      `environment.host + "/api/conditions-for-lure?lureOptionsId=" + lureOptionsId`,
+      `${environment.host}/api/conditions-for-lure?lureOptionsId=${lureOptionsId}`,
```

Comment:
your pr shouldnt touch this file right only lure results?

Reply:
Done